runtime/near-vm-runner/Cargo.toml

```diff
@@ -12,9 +12,14 @@ description = """
 This crate implements the specification of the interface that Near blockchain exposes to the smart contracts.
 """
 
+[target.'cfg(windows)'.dependencies]
+wasmer-runtime = { version = "0.13.1", default-features = true }
```

Comment:
If it is only for standalone then this should be a feature flag in `near-vm-runner` that is being set depending on the target of the standalone crate. Otherwise we might accidentally compile our nearcore for windows with wrong wasm backend.

Reply:
It seems for all package depends on wasmer (include neard), not adding neard here because we currently don't need it. According to @willemneal wasm singlepass backend doesn't build on windows

Reply:
From @willemneal https://github.com/wasmerio/wasmer/issues/1387

Reply:
I'm investigating further with Mike, as he might not be using nightly, which is required for this type of host target dependency and that is why it's failing.

Reply:
That seems unlikely, as we have `rust-toolchain` file, when cargo build cargo would automatically switch to nightly-2020-03-19

Reply:
> It seems for all package depends on wasmer (include neard), not adding neard here because we currently don't need it. 

I am not sure I understand what you mean here. Could you rephrase?

Currently, we cannot build node on Windows, because we cannot build singlepass on Windows. While it is okay to replace singlepass with optimizing compiler in near-vm-standalone, it is not okay to have entire node compiled with optimizing compiler. Therefore, until wasmer singlepass supports Windows we simply cannot have a Windows build of our node.

Reply:
I see. so it's not okay for neard to compile with optimizing compiler. Also @willemneal and I actually cannot build vm-standalone even with optimizing compiler. So we plan to wait until wasmerio/wasmer#1387 is solved and use standalone backend for windows too. If you can ask wasmer to prioritize that issue, would be great.

Reply:
Okay, after they fix their current bugs their next priority is making their stuff compile with stable. Is Windows more or less important than compilation with stable?

Reply:
I think it's less important than compilation stable, and I guess it's not a simple task. Also it would be possible to use docker standalone-vm-runner for windows. @willemneal WDYT?

Reply:
standalone-vm-runner can use optimizing compiler and be compiled to Windows.

Reply:
@nearmax Well we've had issues with it even with turning off the feature in the cargo.toml.  @ailisp docker could work but it'd be a bit heavy.